library/src/main/java/uk/co/samuelwall/materialtaptargetprompt/MaterialTapTargetPrompt.java

```diff
@@ -188,6 +188,7 @@ public void run()
     {
         final ResourceFinder resourceFinder = promptOptions.getResourceFinder();
         mView = new PromptView(resourceFinder.getContext());
+        mView.mPrompt = this;
```

Comment:
Should this reference be set to null in `cleanUpPrompt`?

Reply:
We can, but it isn't necessary. The issue is the prompt holding on to the view, not the other way around.